### DISCUSSION TITLE
Can elementary school students join the Student Developer Pack?

### DISCUSSION THREAD

**Discussion by rudadot** [community/community#53306] [GitHub Education] [answered]
Hello, I am an elementary school student.
I live in Korea, so please understand if I speak English incorrectly. Then, the point is, can elementary school students join the student developer pack?

**Comment by cohhi** [comment]
### Application conditions:
Currently studying at educational institutions such as junior high schools, colleges, universities, home schools, or similar institutions that can award degrees or diplomas
Have a verifiable email address or upload file from the school to prove your current student identity
At least 13 years old

**Comment by cohhi** [comment, marked as answer]
![捕获](https://user-images.githubusercontent.com/71576964/233639560-dec74c47-26c0-449e-af86-ac632838a7fb.PNG)

현재 고등학교, 중학교, 학원, 대학, 가정학교 또는 유사한 교육기관의 학위 또는 졸업장 수여 과정에 재학 중이다.
학교가 게시한 이메일 주소를 확인하거나 현재 학생임을 증명하는 파일을 업로드할 수 있습니다.
적어도 13살.
